src/Decompiler/Analysis/IndirectCallRewriter.cs

```diff
@@ -148,6 +149,28 @@ private void RewriteCall(Statement stm, CallInstruction call)
             Use(adjustRegStm, src);
         }
 
+        private void DefineTrashedIdentifiers(
+            Statement stm,
+            CallInstruction call)
+        {
+            var trashedSids = call.Definitions.Select(d => d.Identifier)
+                .Select(id => ssa.Identifiers[id])
+                .Where(sid => sid.DefStatement == null);
+            foreach (var sid in trashedSids)
+            {
+                DefineTrashedIdentifier(stm, sid);
+            }
+        }
+
+        private void DefineTrashedIdentifier(Statement stm, SsaIdentifier sid)
```

Comment:
Consider renaming this to `DefineUninitializedIdentifier`

Reply:
Done. I have amended last commit.